### PR TITLE
funnyDanielBranch

### PR DESCRIPTION
**Pull request by chengd35**

### CODE REVIEW

**Review by riena42 — APPROVED** [review]
Good work, follows clean architecture and API documentation.

**Comment by anthonymountain** [comment]
I will review in the morning and approve when I can properly refactor views

File: src/main/java/app/AppBuilder.class
Comment (anthonymountain) [inline src/main/java/app/AppBuilder.class:1]: make sure to delete .class files before merging, set your output path to target instead of making output folder in src/main

**Review by anthonymountain — CHANGES_REQUESTED** [review]
make sure to delete .class files before merging, set your output path to designated target folder instead, should already be marked as excluded

**Comment by Descartes-AR** [comment]
> make sure to delete .class files before merging, set your output path to designated target folder instead, should already be marked as excluded, doesn't seem to effect compilation but was very confusing.
> 
> 
> 
> I will manual merge after .class files deleted
##Aren't the class files a result of the compilation, do we really need to delete them?

**Comment by anthonymountain** [comment]
> > make sure to delete .class files before merging, set your output path to designated target folder instead, should already be marked as excluded, doesn't seem to effect compilation but was very confusing.
> > I will manual merge after .class files deleted
> > ##Aren't the class files a result of the compilation, do we really need to delete them?

makes it annoying to go through files, takes up unnecessary space, supposed to go to "target" folder which is marked as excluded so we have unnecessary duplicate files that take up space (my computer has like 20mb of space left i can't run projects after pulling)

**Comment by Descartes-AR** [comment]
aight

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Review by anthonymountain — APPROVED** [review]

**Pull request merged**
